src/components/pages/InstructorBooking/InstructorBookingCard.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { Card } from 'antd';
 import '../../../styles/index.less';
-import { dateConverter } from '../../common/dateHelpers';
-import { timeConverter } from '../../common/timeHelpers';
```

Comment:
I don't believe that these helper functions need to be capitalized since they aren't React components returning JSX.

Reply:
oops, yeah I'll fix those